infra/app/envs/staging/main.tf

```diff
@@ -50,13 +50,18 @@ module "project_config" {
 
 # Add application modules below
 module "app" {
-  source                = "../../env-template"
-  environment_name      = local.environment_name
-  participant_image_tag = var.participant_image_tag
-  staff_image_tag       = var.staff_image_tag
+  source           = "../../env-template"
+  environment_name = local.environment_name
+
+  participant_image_tag             = var.participant_image_tag
+  participant_url                   = "${local.environment_name}.wic-services.org"
+  participant_max_upload_size_bytes = "5242880"
```

Comment:
Should we set these to prod numbers, just so there's a non-prod with the same settings?

Reply:
Good call. Thank you. I'll change that.